src/main/webapp/app/complaints/complaint.service.ts

```diff
@@ -163,6 +163,16 @@ export class ComplaintService implements IComplaintService {
         return this.requestComplaintsFromUrl(url);
     }
 
+    /**
+     * Find all complaints by course id and complaintType.
+     * @param courseId
+     * @param complaintType
+     */
+    findAllByCourseIdAndExamId(courseId: number, examId: number, complaintType: ComplaintType): Observable<EntityResponseTypeArray> {
+        const url = `${this.apiUrl}/courses/${courseId}/exams/${examId}/complaints`;
```

Comment:
Maybe we should remove it if its unused!

Reply:
yea, I removed it. For exams we only have Complaints and no feedbackRequests anyway.